sdk/compute/Azure.ResourceManager.Compute/tests/Scenario/VirtualMachineManagedIdentityTests.cs

```diff
@@ -98,7 +98,7 @@ public async Task UpdateVmIdentityFromNoneToSystem()
             var lro = await collection.CreateOrUpdateAsync(WaitUntil.Completed, vmName, input);
             VirtualMachineResource virtualMachine = lro.Value;
             Assert.AreEqual(vmName, virtualMachine.Data.Name);
-            Assert.Null(virtualMachine.Data.Identity);
+            //Assert.Null(virtualMachine.Data.Identity);
```

Comment:
So if we have to remove this, I need to confirm:
Previously this could pass, meaning that, when we create a VM without any identity specified, we get a response with the `Identity` to be null.
Now if this fails, means this is not null - then which api-version does we have this change?
Does it remain previous behavior in those previous api-versions?

And if returning non-null identity here is expected, we should validate the identity, for instance, it should be `None`
Or if you do not recall this change on the service, does it come from a policy on your subscription?
We need to make sure we know the root cause of this change.

Reply:
Last version release, 2024-03-01, the tests were fine. Now if I create a VM with the version, the identity is populated.

this looks like the result of security protocol for internal subs. Still waiting for confirmation from identity team

Reply:
Previously recorded tests have identity == null , but new ones have it populated so if I replace the line. I have to re-record the tests to have them consistent. How do I re-record an already recorded test ?

Reply:
Since this a test case that validates we could update the identity from None to SystemAssigned, if there is a policy always adding the identity for us, this test is not really doing its job correctly.
Therefore, here I think, we should use the magic tag to disable the policy to apply on this vm instance, and restore the Assert.

You should be able to add the tag by
```
input.Tags.Add(...);
```

Reply:
Tests are re-recorded with the magic tag. 
All original asserts are preserved.